spec/isodoc/pdf_convert_spec.rb

```diff
@@ -1,113 +1,121 @@
 require "spec_helper"
 
 RSpec.describe IsoDoc::BIPM do
-
   it "splits bilingual collection PDF" do
-    FileUtils.rm_f("test.pdf")
-    FileUtils.rm_f("test_fr.pdf")
-    FileUtils.rm_f("test_en.pdf")
```

Comment:
And again, please restore these lines

Reply:
https://github.com/metanorma/metanorma-bipm/pull/115/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R27 - this will help run tests in isolated tmp directory

Reply:
All file operations should be done in a temp directory.